app/javascript/react/screens/App/Settings/screens/ConversionHostsSettings/components/ConversionHostsList.js

```diff
@@ -1,17 +1,61 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { Grid, ListView, Toolbar } from 'patternfly-react';
+import ListViewToolbar from '../../../../common/ListViewToolbar/ListViewToolbar';
 
 const ConversionHostsList = ({ conversionHosts }) => (
-  <React.Fragment>
-    <h2>TODO: render a list view here using the ListViewToolbar pattern we have elsewhere</h2>
-    {conversionHosts.map(conversionHost => (
-      <pre key={conversionHost.id}>{JSON.stringify(conversionHost)}</pre>
-    ))}
-  </React.Fragment>
+  <ListViewToolbar
+    filterTypes={ConversionHostsList.filterTypes}
+    sortFields={ConversionHostsList.sortFields}
+    listItems={conversionHosts}
+  >
+    {({
+      filteredSortedPaginatedListItems,
+      renderFilterControls,
+      renderSortControls,
+      renderActiveFilters,
+      renderPaginationRow
+    }) => (
+      <React.Fragment>
+        <Grid.Row>
+          <Toolbar>
+            {renderFilterControls()}
+            {renderSortControls()}
+            {renderActiveFilters(filteredSortedPaginatedListItems)}
+          </Toolbar>
+        </Grid.Row>
+        <div style={{ overflow: 'auto', paddingBottom: 300, height: '100%' }}>
+          <ListView className="conversion-hosts-list" id="conversion_hosts">
+            {filteredSortedPaginatedListItems.items.map((host, n) => (
+              <ListView.Item key={host.id} heading={host.name} stacked />
```

Comment:
Once we add more details to the list view items, they should probably become their own `ConversionHostListItem` component that we import here.